tools/multi_arch.bzl

```diff
@@ -0,0 +1,27 @@
+"a rule transitioning an oci_image to multiple platforms"
+
+def _multiarch_transition(settings, attr):
+    return [
+        {"//command_line_option:platforms": str(platform)}
+        for platform in attr.platforms
+    ]
+
+multiarch_transition = transition(
+    implementation = _multiarch_transition,
+    inputs = [],
+    outputs = ["//command_line_option:platforms"],
+)
+
+def _impl(ctx):
+    return DefaultInfo(files = depset(ctx.files.image))
```

Comment:
what does this do?

Reply:
It's a "simple" starlark transition implementation. All rules have must return a provider of some kind. This one returns the default provider with information about what should be built in order to fulfill this rule.